app.js

```diff
@@ -3,217 +3,249 @@
 function myGame() {
   var numCorrectResponses = 0;
 
-  // Ask for user's name
-  var userName = prompt('What\'s your name?');
-  alert('Hello ' + userName + '. Let\'s begin!');
-  // Console returns player's name 'userName'
-  console.log('Player\'s name is \"' + userName + '\".')
+  function name() {
```

Comment:
As a stretch goal, if you feel like it, it could be fun to add in some logic for handling the user's name if they don't input one correctly.